COLLADAMaya/include/COLLADAMayaSyntax.h

```diff
@@ -36,7 +36,7 @@ namespace COLLADAMaya
 
     /** Profile specific extra attribute parameters. */
     static const char* PROFILE_COLLADA                              = "COLLADA";
-    static const char* PROFILE_MAYA                                 = "OpenCOLLADAMaya";
+    static const String PROFILE_MAYA                                 = "OpenCOLLADAMaya";
     static const char* PROFILE_MAX                                  = "OpenCOLLADA3dsMax";
```

Comment:
Why is only PROFILE_MAYA switched to String ? Why not switching all of them, or not changing PROFILE_MAYA definition?